dashboard/app/models/script.rb

```diff
@@ -1726,6 +1726,20 @@ def get_assessment_script_levels
     end
   end
 
+  # Determine whether a level in a script should allow pairing based on its
+  # parent level.
+  # TODO: "LevelGroup" levels currently don't have access to their parent level
+  # once they do, simply access the parent level as part of the should_allow_pairing?
+  # check in level.rb
```

Comment:
This functionality will be added by this PR: https://github.com/code-dot-org/code-dot-org/pull/34667

Reply:
✔️